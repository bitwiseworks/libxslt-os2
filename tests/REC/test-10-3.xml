<employees>
  <employee>
    <name>
      <given>Peter</given>
      <family>Mueller</family>
    </name>
  </employee>
  <employee>
    <name>
      <given>Peter</given>
      <family>Möller</family>
    </name>
  </employee>
  <employee>
    <name>
      <given>Peter</given>
      <family>Märtens</family>
    </name>
  </employee>
</employees>
