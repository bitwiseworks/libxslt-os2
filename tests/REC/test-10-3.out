<?xml version="1.0"?>
<ul><li>Peter Märtens</li><li>Peter Möller</li><li>Peter Mueller</li></ul>
